app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def edit; end
+
+  def update
+    @user_form = UserForm.new(User.find(current_user.id), users_params)
+    @user_form.save ? good_update : bad_update
+  end
+
+  def destroy
+    User.destroy(current_user.id)
+    flash[:notice] = t('privacy.destroy')
+    redirect_to root_path
+  end
+
+  private
+
+  def good_update
+    flash[:notice] =
+      t('privacy.good_update',
+        attribute: users_params[:email].present? ? t('privacy.type.email') : t('privacy.type.password'))
+    redirect_to root_path
```

Comment:
pass notice to redirect and why root_path?

Reply:
fixed notice, but in this case i left root_path, because when user change password he also log_out from site